### PR TITLE
N17: Thermite

### PR DESCRIPTION
**Pull request by AngelOnFira**
For #408 :rocket:

### CODE REVIEW

File: content/posts/newsletter-017/index.md [hunk 1]
```diff
@@ -218,6 +218,28 @@ _Discussions:
 [rayon]: https://github.com/rayon-rs/rayon
 [crates.io]: https://crates.io/
 
+### [Termite SIMD]: Melt Your CPU
```
Comment (17cupsofcoffee) [inline content/posts/newsletter-017/index.md:221]:
```suggestion
### [Thermite SIMD]: Melt Your CPU
```

File: content/posts/newsletter-017/index.md [hunk 2]
```diff
@@ -218,6 +218,28 @@ _Discussions:
...
+optimized vectorized math library with many special math functions and
+algorithms, specialized for both single and double precision.
+
+Thermite has been worked on for a little over a month now. The AVX2 backend and
+vectorized math library almost fully implemented. Pre-AVX2/WASM/ARM backends are
+a work in progress. The latest documentation is available
+[here][thermite-documentation].
+
+[thermite-documentation]: https://raygon-renderer.github.io/thermite/
+[Termite SIMD]: https://github.com/raygon-renderer/thermite
```
Comment (17cupsofcoffee) [inline content/posts/newsletter-017/index.md:241]:
```suggestion
[Thermite SIMD]: https://github.com/raygon-renderer/thermite
```

**Review by 17cupsofcoffee — CHANGES_REQUESTED** [review]
Looks good, just a small typo:

File: content/posts/newsletter-017/index.md (hunk 2 above)
Comment (AngelOnFira) [reply content/posts/newsletter-017/index.md:241]: Thanks :sweat_smile:

**Review by 17cupsofcoffee — APPROVED** [review]

**Review by ozkriff — APPROVED** [review]
:+1:

**Comment by ozkriff** [comment]
^ @novacrazy jfyi

**Comment by novacrazy** [comment]
Looks good to me, thanks!

**Pull request merged**